[2025-10-13T08:36:37.491+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:36:37.518+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:36:37.529+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:36:37.530+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:36:37.545+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T08:36:37.553+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=149) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:36:37.556+0000] {standard_task_runner.py:72} INFO - Started process 155 to run task
[2025-10-13T08:36:37.556+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpk0dhmap4']
[2025-10-13T08:36:37.558+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask extract_load_group.load_products
[2025-10-13T08:36:37.626+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:36:37.727+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T08:36:37.728+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:36:41.015+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 2599, in load_table_from_file
    response = self._do_resumable_upload(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 3019, in _do_resumable_upload
    upload, transport = self._initiate_resumable_upload(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 3088, in _initiate_resumable_upload
    upload.initiate(
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/upload.py", line 420, in initiate
    return _request_helpers.wait_and_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/_request_helpers.py", line 155, in wait_and_retry
    response = func()
               ^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/upload.py", line 416, in retriable_request
    self._process_initiate_response(result)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/_upload.py", line 518, in _process_initiate_response
    _helpers.require_status_code(
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/_helpers.py", line 108, in require_status_code
    raise common.InvalidResponse(
google.resumable_media.common.InvalidResponse: ('Request failed with status code', 403, 'Expected one of', 200, 201)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/scripts/load.py", line 22, in load
    job = client.load_table_from_file(
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 2607, in load_table_from_file
    raise exceptions.from_http_response(exc.response)
google.api_core.exceptions.Forbidden: 403 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gcpl-469311/jobs?uploadType=resumable: Access Denied: Project gcpl-469311: User does not have bigquery.jobs.create permission in project gcpl-469311.
[2025-10-13T08:36:41.035+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T083637, end_date=20251013T083641
[2025-10-13T08:36:41.051+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:36:41.052+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 13 for task extract_load_group.load_products (403 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gcpl-469311/jobs?uploadType=resumable: Access Denied: Project gcpl-469311: User does not have bigquery.jobs.create permission in project gcpl-469311.; 155)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 2599, in load_table_from_file
    response = self._do_resumable_upload(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 3019, in _do_resumable_upload
    upload, transport = self._initiate_resumable_upload(
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 3088, in _initiate_resumable_upload
    upload.initiate(
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/upload.py", line 420, in initiate
    return _request_helpers.wait_and_retry(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/_request_helpers.py", line 155, in wait_and_retry
    response = func()
               ^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/requests/upload.py", line 416, in retriable_request
    self._process_initiate_response(result)
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/_upload.py", line 518, in _process_initiate_response
    _helpers.require_status_code(
  File "/home/airflow/.local/lib/python3.12/site-packages/google/resumable_media/_helpers.py", line 108, in require_status_code
    raise common.InvalidResponse(
google.resumable_media.common.InvalidResponse: ('Request failed with status code', 403, 'Expected one of', 200, 201)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/scripts/load.py", line 22, in load
    job = client.load_table_from_file(
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/google/cloud/bigquery/client.py", line 2607, in load_table_from_file
    raise exceptions.from_http_response(exc.response)
google.api_core.exceptions.Forbidden: 403 POST https://bigquery.googleapis.com/upload/bigquery/v2/projects/gcpl-469311/jobs?uploadType=resumable: Access Denied: Project gcpl-469311: User does not have bigquery.jobs.create permission in project gcpl-469311.
[2025-10-13T08:36:41.104+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-10-13T08:36:41.118+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T08:44:45.212+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:44:45.240+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:44:45.249+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:44:45.249+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:44:45.264+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T08:44:45.273+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=500) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:44:45.275+0000] {standard_task_runner.py:72} INFO - Started process 504 to run task
[2025-10-13T08:44:45.275+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpn1_f1oxf']
[2025-10-13T08:44:45.277+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask extract_load_group.load_products
[2025-10-13T08:44:45.351+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:44:45.450+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T08:44:45.451+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:44:58.358+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-13T08:44:58.359+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T08:44:58.373+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:44:58.373+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T084445, end_date=20251013T084458
[2025-10-13T08:44:58.419+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T08:44:58.440+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T08:44:58.442+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T08:54:00.974+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:54:01.000+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:54:01.009+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T08:54:01.010+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:54:01.025+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T08:54:01.032+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=692) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:54:01.034+0000] {standard_task_runner.py:72} INFO - Started process 693 to run task
[2025-10-13T08:54:01.035+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpd_xzzd53']
[2025-10-13T08:54:01.036+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask extract_load_group.load_products
[2025-10-13T08:54:01.095+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:54:01.190+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T08:54:01.192+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:54:21.658+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-13T08:54:21.659+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T08:54:21.669+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:54:21.670+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T085401, end_date=20251013T085421
[2025-10-13T08:54:21.722+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T08:54:21.740+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T08:54:21.743+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T09:05:36.773+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T09:05:36.796+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:05:36.806+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:05:36.807+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T09:05:36.823+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T09:05:36.831+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1026) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T09:05:36.834+0000] {standard_task_runner.py:72} INFO - Started process 1027 to run task
[2025-10-13T09:05:36.834+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp8bbj3t7r']
[2025-10-13T09:05:36.836+0000] {standard_task_runner.py:105} INFO - Job 151: Subtask extract_load_group.load_products
[2025-10-13T09:05:36.899+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T09:05:36.990+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T09:05:36.991+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T09:05:41.460+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-13T09:05:41.460+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T09:05:41.472+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T09:05:41.473+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T090536, end_date=20251013T090541
[2025-10-13T09:05:41.509+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T09:05:41.536+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T09:05:41.539+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T09:15:17.812+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T09:15:17.834+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:15:17.843+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:15:17.844+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T09:15:17.860+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T09:15:17.868+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1186) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T09:15:17.869+0000] {standard_task_runner.py:72} INFO - Started process 1193 to run task
[2025-10-13T09:15:17.869+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmppi8eva8d']
[2025-10-13T09:15:17.871+0000] {standard_task_runner.py:105} INFO - Job 161: Subtask extract_load_group.load_products
[2025-10-13T09:15:17.931+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T09:15:18.025+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T09:15:18.026+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T09:15:23.001+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-13T09:15:23.002+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T09:15:23.016+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T09:15:23.017+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T091517, end_date=20251013T091523
[2025-10-13T09:15:23.066+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T09:15:23.091+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T09:15:23.093+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T09:20:34.216+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T09:20:34.246+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:20:34.260+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-13T09:20:34.260+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T09:20:34.280+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-13T09:20:34.291+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1286) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T09:20:34.294+0000] {standard_task_runner.py:72} INFO - Started process 1294 to run task
[2025-10-13T09:20:34.294+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpqpr5qb7z']
[2025-10-13T09:20:34.296+0000] {standard_task_runner.py:105} INFO - Job 173: Subtask extract_load_group.load_products
[2025-10-13T09:20:34.380+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T09:20:34.503+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-13T09:20:34.505+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T09:20:45.352+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-13T09:20:45.353+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T09:20:45.364+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T09:20:45.365+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251013T092034, end_date=20251013T092045
[2025-10-13T09:20:45.398+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T09:20:45.423+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T09:20:45.425+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:15:41.926+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:15:41.947+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:15:41.957+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:15:41.957+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:15:41.972+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:15:41.981+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:15:41.983+0000] {standard_task_runner.py:72} INFO - Started process 132 to run task
[2025-10-14T04:15:41.984+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '244', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp_kyga9j8']
[2025-10-14T04:15:41.985+0000] {standard_task_runner.py:105} INFO - Job 244: Subtask extract_load_group.load_products
[2025-10-14T04:15:42.048+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:15:42.141+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:15:42.142+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:15:48.674+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:15:48.675+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:15:48.688+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:15:48.688+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T041541, end_date=20251014T041548
[2025-10-14T04:15:48.747+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:15:48.776+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:15:48.779+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:20:40.577+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:20:40.617+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:20:40.635+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:20:40.637+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:20:40.672+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:20:40.683+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:20:40.686+0000] {standard_task_runner.py:72} INFO - Started process 275 to run task
[2025-10-14T04:20:40.687+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp8_0oad7v']
[2025-10-14T04:20:40.690+0000] {standard_task_runner.py:105} INFO - Job 266: Subtask extract_load_group.load_products
[2025-10-14T04:20:40.759+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:20:40.852+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:20:40.853+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:20:44.786+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:20:44.787+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:20:44.801+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:20:44.802+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T042040, end_date=20251014T042044
[2025-10-14T04:20:44.836+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:20:44.862+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:20:44.864+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:22:15.459+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:22:15.481+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:22:15.491+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:22:15.492+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:22:15.512+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:22:15.519+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=501) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:22:15.522+0000] {standard_task_runner.py:72} INFO - Started process 514 to run task
[2025-10-14T04:22:15.521+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp4ywkpxmf']
[2025-10-14T04:22:15.523+0000] {standard_task_runner.py:105} INFO - Job 327: Subtask extract_load_group.load_products
[2025-10-14T04:22:15.587+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:22:15.689+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:22:15.690+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:22:21.621+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:22:21.622+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:22:21.636+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:22:21.637+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T042215, end_date=20251014T042221
[2025-10-14T04:22:21.677+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:22:21.703+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:22:21.705+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:26:08.713+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:26:08.739+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:26:08.750+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:26:08.751+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:26:08.766+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:26:08.774+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=647) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:26:08.776+0000] {standard_task_runner.py:72} INFO - Started process 650 to run task
[2025-10-14T04:26:08.777+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '348', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpcde0ehd3']
[2025-10-14T04:26:08.778+0000] {standard_task_runner.py:105} INFO - Job 348: Subtask extract_load_group.load_products
[2025-10-14T04:26:08.846+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:26:08.962+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:26:08.963+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:26:14.934+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:26:14.934+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:26:14.947+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:26:14.947+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T042608, end_date=20251014T042614
[2025-10-14T04:26:14.980+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:26:15.007+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:26:15.008+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:28:12.372+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:28:12.398+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:28:12.408+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:28:12.409+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:28:12.423+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:28:12.430+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=753) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:28:12.432+0000] {standard_task_runner.py:72} INFO - Started process 756 to run task
[2025-10-14T04:28:12.433+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '372', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp4oqtvc96']
[2025-10-14T04:28:12.435+0000] {standard_task_runner.py:105} INFO - Job 372: Subtask extract_load_group.load_products
[2025-10-14T04:28:12.502+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:28:12.609+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:28:12.610+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:28:18.081+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:28:18.082+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:28:18.094+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:28:18.094+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T042812, end_date=20251014T042818
[2025-10-14T04:28:18.144+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:28:18.166+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:28:18.168+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:31:20.803+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:31:20.833+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:31:20.843+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-10-14T04:31:20.844+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:31:20.860+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_products> on 2025-09-01 00:00:00+00:00
[2025-10-14T04:31:20.868+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1053) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:31:20.870+0000] {standard_task_runner.py:72} INFO - Started process 1057 to run task
[2025-10-14T04:31:20.870+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '445', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpuwaataie']
[2025-10-14T04:31:20.871+0000] {standard_task_runner.py:105} INFO - Job 445: Subtask extract_load_group.load_products
[2025-10-14T04:31:20.942+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:31:21.077+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-10-14T04:31:21.079+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:31:25.153+0000] {load.py:30} INFO - Loaded 1000 rows and 4 columns to gcpl-469311.datalabs_techtest.stg_products
[2025-10-14T04:31:25.154+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:31:25.168+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:31:25.168+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20251014T043120, end_date=20251014T043125
[2025-10-14T04:31:25.221+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:31:25.232+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
